app/assets/javascripts/fae/form/fae_chosen.js

```diff
@@ -21,6 +21,23 @@
       }
 
       $this.chosen(settings);
+
+      // add handling for bottom of viewport / drop-up support
+      $this.on('chosen:showing_dropdown', function(event, params) {
+        var chosen_container = $(event.target ).next('.chosen-container');
```

Comment:
`$` was not in place for surrounding jQuery variables, so I avoided introducing it. Will refactor to include